linkerd/app/inbound/src/http/mod.rs

```diff
@@ -133,6 +133,7 @@ where
 
         // Creates HTTP clients for each inbound port & HTTP settings.
         let endpoint = connect
+            .push(svc::stack::BoxFuture::layer()) // Makes the response future `Unpin`.
```

Comment:
I moved this from `to_tcp_connect` because it's really the http_router that needs this behavior -- but no other users of the tcp connect stack needs this. This simplifies type constraints.

Reply:
Talked off GitHub about this but makes sense 👍